examples/java/override_methods/NoOverrideTargetParam.java

```diff
@@ -0,0 +1,30 @@
+class A 
+{
+    int i;
+  
+    //@ predicate valid() = this.i |-> ?m_i;
+      
+    public int getI()
+    //@ requires valid();
+    //@ ensures valid();
+    {
+        return i;
+    }
+}
+
+
+class B extends A //~
+{
+    int j;
+    
+    //@ predicate valid() = this.valid(A.class)() &*& this.j |-> ?m_j;
```

Comment:
Define `B.valid` in such a way that it does *not* imply `A.valid` (and define `A.valid` as `false` and write `A.getI()` so that it crashes)! For this example as it is now, the fact that VeriFast does not accept it is unfortunate; show an example where it is essential that VeriFast does not accept it.